tests/test_modeling_flax_common.py

```diff
@@ -310,3 +312,84 @@ def check_hidden_states_output(inputs_dict, config, model_class):
             config.output_hidden_states = True
 
             check_hidden_states_output(inputs_dict, config, model_class)
+
+    def test_attention_outputs(self):
+
+        from .test_modeling_flax_gpt2 import FlaxGPT2ModelTest
+
+        if isinstance(self, FlaxGPT2ModelTest):
```

Comment:
We should also allow GPT2 to return `attention_outputs` - I think we can remove this :-)

Reply:
Yeah, this is temporary, will remove this after adding the same changes for GPT2.